core/playlist.py

```diff
@@ -21,6 +21,7 @@ def __init__(self, link_of_playlist):
         self.pl_response = None
         
         self.vid_ids = None
+        self.youtube = None
```

Comment:
self.youtube must not be initialized with None as it is the one that uses build method of the googleapiclient.discovery module. 
It should be:
`self.youtube = build('youtube', 'v3', developerKey=self.api_key)`